src/passkey_child/passkey_child.h

```diff
@@ -183,4 +184,59 @@ errno_t
 print_credentials(const struct passkey_data *data,
                   const fido_cred_t *const cred);
 
+/**
+ * @brief Format es256 public key to PEM format
+ *
+ * @param[in] mem_ctx Memory context
+ * @param[in] ptr Public key pointer
+ * @param[in] ptr_len Public key lenth
```

Comment:
lenth -> length

Reply:
Fixed.

Reply:
Sorry I didn't mention explicitly, but the same mistype is in comments of `rs256_pubkey_to_evp_pkey()` and `eddsa_pubkey_to_evp_pkey()`

Reply:
I forgot to check those too